themes/cardinal_service/lib/scss/spotlight/spotlight.scss

```diff
@@ -0,0 +1,168 @@
+@charset "UTF-8";
+
+.spotlight-all-content-container {
+	display: flex;
+	align-items: center;
+	margin: 0 auto;
+	padding: 70px 0 0;
+
+	.image {
+		display: flex;
+		justify-content: flex-end;
+	}
+
+	img {
+		border-radius: 50%;
+		height: auto;
+		max-width: 100%;
+		vertical-align: middle;
+	}
+}
+
+.su-spotlight-student-name {
+	h2 {
+		font-size: 2.4em;
+		margin: 0 0 20px;
+	}
+}
+
+.node-su-spotlight-su-spotlight-graduation-year,
+.node-su-spotlight-su-spotlight-major {
+	display: inline-block;
+	font-size: 1.2em;
+	margin: 0;
+}
+
+.content {
+	margin-bottom: 100px;
+}
+
+.su-spotlight {
+	&.body {
+		padding: 2em 0 0 0;
+		width: 65%;
+		margin: 0 auto;
+	}
+}
+
+.su-spotlight-quote {
+	font-size: 1.6em;
+	font-weight: 400;
+	line-height: 1.5em;
+
+	&::before {
+		content: open-quote;
+	}
+
+	&::after {
+		content: close-quote;
+	}
+}
+
+.su-spotlight-body {
+	font-size: 1em;
+}
+
+@media (min-width: 1900px) {
```

Comment:
you can use decanter's `@include` statements instead of using media queries. see https://github.com/SU-SWS/stanford_profile/blob/e9172d59109a9e333fa61beeedb63a289ac12dba/modules/stanford_profile_styles/lib/scss/node-types/stanford_page.scss#L22

That will make this like something like 
```
	.spotlight-all-content-container {
            @include grid-media-max('md') {
		width: 50%;
            }
	}
```
And then it can actually be nested under the same selector at the top of the file.

Reply:
Decanter has been imported in the `_config.scss` and the breakpoints have been added (except for `@media (max-width: 1900px)` and `@media (min-width: 1900px)`.